templates/Symbiote/GridFieldExtensions/GridFieldConfigurablePaginator.ss

```diff
@@ -1,7 +1,7 @@
 <tr>
-    <td class="bottom-all" colspan="$Colspan">
+    <td class="grid-field__paginator bottom-all" colspan="$Colspan">
         <span class="pagination-page-size">
-            <%t GridFieldConfigurablePaginator.SHOW 'Show' %>
+            <%t Symbiote\\GridFieldExtensions\\GridFieldConfigurablePaginator.SHOW 'Show' is 'Verb. Example: Show 1 of 2' %>
```

Comment:
what is this `is` syntax? I've not seen it before.

Reply:
I can't find a documentation reference for it, but it's basically describing what the translation is for (developer) reference

Reply:
Kinda like a comment. Cool, just wanted to know that it wasn't some kind of syntax issue :)